nifi-provenance-reporting-tasks/src/main/java/io/egm/nifi/reporting/EmailProvenanceReporter.java

```diff
@@ -212,10 +229,14 @@ public PasswordAuthentication getPasswordAuthentication() {
      * @return an InternetAddress[] parsed from the supplied property
      * @throws AddressException if the property cannot be parsed to a valid InternetAddress[]
      */
-    private InternetAddress[] toInetAddresses(final ReportingContext context, PropertyDescriptor propertyDescriptor)
+    private InternetAddress[] toInetAddresses(final ReportingContext context, final Map<String, Object> event, PropertyDescriptor propertyDescriptor)
```

Comment:
take care of the error message in case the value of the "specific recipient" attribute is not a valid email address